package.json

```diff
@@ -161,5 +161,8 @@
       "<rootDir>/node_modules/csstree-validator"
     ]
   },
-  "packageManager": "yarn@3.4.1"
+  "packageManager": "yarn@3.4.1",
+  "devDependencies": {
+    "@faker-js/faker": "^7.6.0"
```

Comment:
Can you move this to dependencies so it aligns with everything else?

Reply:
Sure thing. Just curious, is there a particular reason we don't do deps vs devdeps?